k8s/prometheus/manifest/grafana-statefulset.yaml

```diff
@@ -13,6 +13,8 @@ spec:
     matchLabels: *Labels
   template:
     metadata:
+      annotations:
+        prometheus.io/scrape: 'true'
```

Comment:
prometheus.io/port also should be defined to make it works.

Reply:
Hmm… I believe that setting `prometheus.io/scrape: 'true'` turns on scraping of all of the pod's endpoints (which is only 1, in Grafana's case here).

Grafana's metrics are exposed on the same port as the service (Grafana's web UI), so I think this is sufficient.

(I'm running without the `port` annotation in production today, and it works).

Am I missing something?